src/fromager/commands/download_sequence.py

```diff
@@ -23,13 +26,19 @@
     default=False,
     is_flag=True,
 )
+@click.option(
+    "--num-threads",
+    default=os.cpu_count(),
```

Comment:
mypy didn't like it when sched_getaffinity() did not exist so I'm dropping that for now so we can land this.